unstructured/ingest/connector/jira.py

```diff
@@ -323,18 +296,42 @@ def _output_filename(self):
             Path(self.partition_config.output_dir) / self.grouping_folder_name / output_file
         ).resolve()
 
+    def update_source_metadata(self, **kwargs) -> None:
+        exists = bool(self.issue)
+        if not exists:
+            self.source_metadata = SourceMetadata(
+                exists=exists,
+            )
+            return
+        version = None
+        if len(self.parsed_fields["versions"]) > 1:
+            version = self.parsed_fields["versions"][-1].get("id")
+        self.source_metadata = SourceMetadata(
+            date_created=datetime.strptime(
+                self.parsed_fields["created"],
+                "%Y-%m-%dT%H:%M:%S.%f%z",
+            ).isoformat(),
+            date_modified=datetime.strptime(
+                self.parsed_fields["updated"],
+                "%Y-%m-%dT%H:%M:%S.%f%z",
+            ).isoformat(),
+            version=version,
```

Comment:
I don't see a version in any of our expected outputs. how are we validating this?

Reply:
got it, so in theory this could exist, but doesn't in our test content.

Reply:
Is this field a "(~release) version" for a project, or a version for the issue content (say, before and after an edit to the issue content)?

Reply:
If the field is a version for the issue content itself, testing this might need updating documents at the source to introduce a second version to some issues

I think it'll be worth it to do those updates in our test jira org and test if it's working as intended (mainly since we're introducing the version field with this PR)

At best, it'll only require a change in a document in our test jira org and rerunning the CI. At worst, it will save us some debugging and a patch later on